internal/tui/model_list.go

```diff
@@ -163,14 +263,14 @@ func (m ListModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 					return errorMsg{errors.New("unable to select an item")}
 				}
 
-				m.selected = string(selected.suggestion)
+				m.selected = selected
 
 				cwd, err := os.Getwd()
 				if err != nil {
 					return errorMsg{err}
 				}
 
-				cmdSlice := []string{"git", "checkout", "-b", m.selected}
+				cmdSlice := []string{"git", "checkout", "-b", m.selected.branchName}
```

Comment:
```suggestion
				cmdSlice := []string{"git", "checkout", "-b", m.selected.Suggestion()}
```

Careful here, we don't want to use the branch name, we want to use the suggested branch name (e.g. without the prefix)

Reply:
good catch thank you.